### ISSUE TITLE
Extending hard drives

### ISSUE THREAD

**Issue by MattDHill** [Start9Labs/embassy-os#2060]

**Comment by leemurdock** [comment]
Does this request make it possible to access an external hard drive using StartOS?

Has any work been done for this issue? TIA @dr-bonez @MattDHill

**Comment by MattDHill** [comment]
Yes, this feature would make.iy possible to mount additional drives to StartOS and use them for service data.

It is not currently being worked on, planned for after the big 040 release.

**Comment by king-11** [comment]
Hey @MattDHill is there something going on it now I actually just want to attach a hard disk and use it as data drive without formatting it

**Comment by MattDHill** [comment]
I'm not sure exactly what you're asking. 0.4.0 will not have the ability to add _extra_ data drives, but it's always been possible to use one drive for StartOS and another for service data.

**Comment by king-11** [comment]
But it always formats the drive which does seem unnecessary at times maybe I want to migrate data from somewhere not formatting the service data seems like a useful thing to have. fyi I migrated from umbrel to start9 and during migration lost a lot of data due to some actions I took mistakenly while copying stuff.
